features/gb/london-cycles.geojson

```diff
@@ -3,6 +3,7 @@
   "features": [
     {
       "type": "Feature",
+      "id": "london-cycles.geojson",
```

Comment:
Would `npm run build` add these? Could it be done on Travis before it tries to validate if so?

Nope, they're still just failing! :confused:

Reply:
I've removed the .geojson file and replaced it with a single GPS location which will include a 25km radius of London

Reply:
That brought up a new issue covered in #4776 so I've switched back.